testing/trino-product-tests/src/main/java/io/trino/tests/product/hive/TestHiveTransactionalTable.java

```diff
@@ -832,6 +832,28 @@ public void testDeleteAllRowsInPartition()
         });
     }
 
+    @Test(groups = HIVE_TRANSACTIONAL, timeOut = TEST_TIMEOUT)
+    public void testDeleteAllRowsUnpartitioned()
+    {
+        withTemporaryTable("delete_all_rows", true, false, NONE, tableName -> {
+            onTrino().executeQuery(format("CREATE TABLE %s (id INT) WITH (transactional = true)", tableName));
+
+            log.info("About to insert");
+            onTrino().executeQuery(format("INSERT INTO %s VALUES (1), (2), (3)", tableName));
+
+            log.info("About to delete one row");
+            onTrino().executeQuery(format("DELETE FROM %s WHERE id = 2", tableName));
+
+            log.info("About to verify");
+            verifySelectForTrinoAndHive("SELECT * FROM " + tableName, "TRUE", row(1), row(3));
```

Comment:
nit: lowercase `true`

Reply:
verification of this stage is a bit redundant (we already have some tests for delete), especially if we run this on Hive too. remove, or keep only on Trino.

Reply:
Changed to test only on Trino.